specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/StorageAccountSetBlobInventoryPolicy.json

```diff
@@ -0,0 +1,78 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res7687",
+    "accountName": "sto9699",
+    "blobInventoryPolicyName": "default",
+    "api-version": "2020-08-01-preview",
+    "monitor": "true",
+    "properties": {
+      "properties": {
+        "policy": {
+          "enabled": true,
+          "type": "Inventory",
+          "destination": "containerName",
+          "rules": [
```

Comment:
>rules [](start = 11, length = 5)

How many rules would you see typically? What would be the max number of rules?

Also, will this ever need to become a nested resource type under inventoryPolicies?

Reply:
A rule is not a separate resource and the model will not change, a blob inventory policy is defined by a set of rules. These rules for preview have a limit of 10 but this constraint will be lifted in the following months on GA. This is controlled on the server-side, we do not want to have a breaking change here

Reply:
Makes sense.